tests/test_main.py

```diff
@@ -0,0 +1,109 @@
+import pytest
+import os
+from gha_runner.__main__ import (
+    parse_aws_params,
+    env_parse_helper,
+)
+
+
+def test_env_parse_helper():
+    base: dict[str, str] = {}
+    assert env_parse_helper(base, "TEST_ENV_VAR_GHA_RUNNER", "test") == base
+    os.environ["TEST_ENV_VAR_GHA_RUNNER"] = "test"
+    assert env_parse_helper(base, "TEST_ENV_VAR_GHA_RUNNER", "test") == {
+        "test": "test"
+    }
+
+
+def test_parse_aws_params():
```

Comment:
this works, but as a testing style, this might be an opportunity for `pytest.mark.parametrize`. That said, you would also need a test that checks for the results with multiple things set (to show it is additive).

Reply:
Fixed in [91bc8e0](https://github.com/omsf-eco-infra/gha-runner/pull/8/commits/91bc8e023f7e4421945706c4c35f16a13fc60dc2).